docs/migrations/v4-to-v5.md

```diff
@@ -0,0 +1,390 @@
+   paramSpace paramtypes.Subspace, 
+   ck types.ClientKeeper
+) Keeper
+```
+
+### ICS04 - Channel 
+
+The function `NewPacketId` in `modules/core/04-channel/types` has been renamed to `NewPacketID`:
+
+```diff
+-  func NewPacketId(
++  func NewPacketID(
+  portID, 
+  channelID string, 
+  seq uint64
+) PacketId 
+```
+
+The `key` parameter of the `NewKeeper` function in `modules/core/04-channel/keeper` is now of type `storetypes.StoreKey` (where `storetypes` is an import alias for `"github.com/cosmos/cosmos-sdk/store/types"`):
+
+```diff
+func NewKeeper(
+   cdc codec.BinaryCodec,
+-  key sdk.StoreKey,
++  key storetypes.StoreKey,
+   clientKeeper types.ClientKeeper,
+   connectionKeeper types.ConnectionKeeper,
+   portKeeper types.PortKeeper, 
+   scopedKeeper capabilitykeeper.ScopedKeeper,
+) Keeper 
+```
+
+### ICS20 - Transfer
+
+The `key` parameter of the `NewKeeper` function in `modules/apps/transfer/keeper` is now of type `storetypes.StoreKey` (where `storetypes` is an import alias for `"github.com/cosmos/cosmos-sdk/store/types"`):
+
+```diff
+func NewKeeper(
+   cdc codec.BinaryCodec,
+-  key sdk.StoreKey,
++  key storetypes.StoreKey, 
+   paramSpace paramtypes.Subspace,
+   ics4Wrapper types.ICS4Wrapper, 
+   channelKeeper types.ChannelKeeper,
+   portKeeper types.PortKeeper,
+   authKeeper types.AccountKeeper,
+   bankKeeper types.BankKeeper,
+   scopedKeeper capabilitykeeper.ScopedKeeper,
+) Keeper
+```
+
+The `amount` parameter of function `GetTransferCoin` in `modules/apps/transfer/types` is now of type `math.Int` (`"cosmossdk.io/math"`):
+
+```diff
+func GetTransferCoin(
+   portID, channelID, baseDenom string,
+-  amount sdk.Int
++  amount math.Int
+) sdk.Coin
+```
+
+The `RegisterRESTRoutes` function in `modules/apps/transfer` has been removed.
+
+### ICS27 - Interchain Accounts
+
+The `key` parameter of the `NewKeeper` functions in 
+
+- `modules/apps/27-interchain-accounts/controller/keeper` 
+- and `modules/apps/27-interchain-accounts/host/keeper` 
+
+ is now of type `storetypes.StoreKey` (where `storetypes` is an import alias for `"github.com/cosmos/cosmos-sdk/store/types"`):
+
+```diff
+// NewKeeper creates a new interchain accounts controller Keeper instance
+func NewKeeper(
+   cdc codec.BinaryCodec,
+-  key sdk.StoreKey,
++  key storetypes.StoreKey,
+   paramSpace paramtypes.Subspace,
+   ics4Wrapper icatypes.ICS4Wrapper,
+   channelKeeper icatypes.ChannelKeeper,
+   portKeeper icatypes.PortKeeper,
+   scopedKeeper capabilitykeeper.ScopedKeeper,
+   msgRouter *baseapp.MsgServiceRouter,
+) Keeper  
+```
+
+```diff
+// NewKeeper creates a new interchain accounts host Keeper instance
+func NewKeeper(
+   cdc codec.BinaryCodec,
+-  key sdk.StoreKey,
++  key storetypes.StoreKey,
+   paramSpace paramtypes.Subspace,
+   channelKeeper icatypes.ChannelKeeper,
+   portKeeper icatypes.PortKeeper,
+   accountKeeper icatypes.AccountKeeper,
+   scopedKeeper capabilitykeeper.ScopedKeeper,
+   msgRouter *baseapp.MsgServiceRouter,
```

Comment:
Should we update this with the new interface definition from #2058?

Reply:
Updated.